### ISSUE TITLE
After registratiin or manual user creation no login possible

### ISSUE THREAD

**Issue by rschenk0** [antedebaas/DMARC-Reports#86]
After manually creating a user as "reports@domain.de" with password and flagged verified for a single domain "role" no login with this user ist possible. PHP Logs Syntax error:

```
NOTICE: PHP message: [critical] Uncaught Exception: [Syntax Error] line 0, col 11: Error: Expected IdentificationVariable | ScalarExpression | AggregateExpression | FunctionDeclaration | "(" Subselect ")" | CaseExpression, got 'FROM'
``` 
After that I deleted the user and used "Register" to register a user and verify the user by an admin. Same error.

**Comment by antedebaas** [comment]
thank you for reporting.
i can confirm the issue and will be working on a fix.

**Comment by antedebaas** [comment]
should be fixed in 1.12.1
please remove all domains rights from the user(s), then update, then re add the user domain rights
